src/actions/guides/frontend/internationalization.cr

```diff
@@ -0,0 +1,545 @@
+        sign_up_failure: "Couldn't sign you up"
+        sign_out: "Sign Out"
+        signed_out: "You have been signed out"
+        pwd_update: "Update Password"
+        pwd_update_success: "Your password has been reset"
+        pwd_reset: "Password Reset"
+        pwd_reset_request: "Reset your Password"
+        pwd_reset_req_success: "You should receive an email on how to reset your password shortly"
+      auth_token:
+        not_authenticated: "Not Authenticated."
+        invalid: "The provided authentication token was incorrect."
+        missing: "An authentication token is required. Please include a token in an 'auth_token' param or 'Authorization' header."
+      default:
+        page_name: Welcome
+      error:
+        title: "Something went wrong"
+        try_home: "Try heading back to home"
+        locked_out: "Locked-out"
+        auth_incorrect: "is wrong"
+        form_not_valid: "It looks like the form is not valid"
+        not_in_system: "is not in our system"
+      user:
+        email: E-Mail
+        profile: Profile
+        next: "Next, you may want to"
+        auth_guides: "Check out the 'Authentication Guides'"
+        modify_page: "Modify this page"
+        after_signin: "Change where you go after sign in"
+        preferred_language: "Preferred Language"
+    ```
+
+    ## Step 3 - Configure i18n within Lucky
+
+    The i18n needs to know the location of your language files.
+    > NOTE: This config file is only executed at load, so all language changes require a server restart.
+
+    ```
+    # config/i18n.cr
+    I18n.backend = I18n::Backends::YAML.new.tap do |backend|
+      backend.load_paths << Dir.current + "/config/locales"
+      backend.load
+    end
+    ```
+
+    ## Step 4 - Add 'lang' to users table
+
+    This setup will assocatiate a language key with each user this language key is used when displaying information.
+
+    Generate a migration using:
+    ```
+    lucky gen.migration AddLanguageToUser
+    ```
+
+    Edit the new migration file in `db/migrations/`:
+    ```
+    # db/migrations/#{Time.utc.to_s("%Y%m%d%H%I%S")}_add_language_to_user.cr
+    class AddLanguageToUser::V20191228100116 < Avram::Migrator::Migration::V1
+      def migrate
+        alter table_for(User) do
+          add lang : String, default: "en"  # the appropriate default lang key
+        end
+      end
+
+      def rollback
+        alter table_for(User) do
+          remove :lang
+        end
+      end
+    end
+    ```
+
+    Of course migrate
+    ```
+    lucky db.migrate
+    ```
+
+    ## Step 5 - Add lang column to User model
+    ```
+    # src/models/user.cr
+    class User < BaseModel
+      # ...
+      table do
+        column lang : String
+        # ...
+      end
+      # ...
+    end
+    ```
+
+    ## Step 6 - Create a Translator module
+
+    First create a location to extend your lucky system (I suggest `mixins`):
+
+    ```
+    $ mkdir src/mixins
+    $ touch src/mixins/translator.cr
```

Comment:
I'd probably recommend putting it directly in `src/translator.cr` since the mixin is global. That's typically where I'd put more "general purpose" code that isn't tied to actions, pages, etc.

Reply:
Ok - I can change it back.

Reply:
Reverted.